homeassistant/components/bittrex/__init__.py

```diff
@@ -0,0 +1,93 @@
+"""Gather the market details from Bittrex."""
+import logging
+from typing import Dict
+
+from aiobittrexapi import Bittrex
+from aiobittrexapi.errors import (
+    BittrexApiError,
+    BittrexInvalidAuthentication,
+    BittrexResponseError,
+)
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_API_KEY
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
+
+from .const import CONF_API_SECRET, CONF_MARKETS, DOMAIN, SCAN_INTERVAL
+
+_LOGGER = logging.getLogger(__name__)
+
+CONFIG_SCHEMA = vol.Schema(
```

Comment:
I think this CONFIG_SCHEMA declaration is an unneeded, legacy artifact that this new integration would never have needed since it is starting as something that always used config flow.

Reply:
Thanks!